wire-schema-tests/build.gradle.kts

```diff
@@ -61,6 +69,43 @@ kotlin {
         }
       }
     }
+    if (System.getProperty("knative", "true").toBoolean()) {
+      val nativeMain by creating {
+        dependsOn(commonMain)
+      }
+      val nativeTest by creating {
+        dependsOn(commonTest)
+      }
+
+      val iosX64Main by getting
+      val iosArm64Main by getting
+      val iosSimulatorArm64Main by getting
+      val linuxX64Main by getting
+      val macosX64Main by getting
+      val macosArm64Main by getting
+      val mingwX64Main by getting
+      val tvosX64Main by getting
+      val tvosArm64Main by getting
+      val tvosSimulatorArm64Main by getting
+      val iosX64Test by getting
+      val iosArm64Test by getting
+      val iosSimulatorArm64Test by getting
+      val linuxX64Test by getting
+      val macosX64Test by getting
+      val macosArm64Test by getting
+      val mingwX64Test by getting
+      val tvosX64Test by getting
+      val tvosArm64Test by getting
+      val tvosSimulatorArm64Test by getting
+
+      for (it in listOf(iosX64Main, iosArm64Main, iosSimulatorArm64Main, linuxX64Main, macosX64Main, macosArm64Main, mingwX64Main, tvosX64Main, tvosArm64Main, tvosSimulatorArm64Main)) {
```

Comment:
So looking forward to this thing:
https://kotlinlang.org/docs/whatsnew-eap.html#template-for-configuring-multiplatform-projects

Reply:
I think @chrisbanes added it in Circuit. What bothers me with the code in both before/after cases is that I don't know a good way to check everything is working fine